test_main.py

```
@@ -54,21 +56,88 @@
+    mock_get.return_value = mock_response
 
-## Header 1
+    # Test the JSON output format
+    try:
+        # Try to use the Rust implementation first
+        from markdown_lab_rs import OutputFormat, convert_html
 
-Paragraph 1
+        # Convert to JSON
+        json_content = convert_html(
+            mock_response.text, "http://example.com", OutputFormat.JSON
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)